sphinx/builders/html.py

```diff
@@ -1526,9 +1561,14 @@ def setup(app):
     app.add_config_value('html_search_scorer', '', None)
     app.add_config_value('html_scaled_image_link', True, 'html')
     app.add_config_value('html_experimental_html5_writer', None, 'html')
+    app.add_config_value('html_math_renderer', None, 'env')
 
     # event handlers
     app.connect('config-inited', convert_html_css_files)
+    app.connect('builder-inited', validate_math_renderer)
+
+    # load default math renderer
+    app.setup_extension('sphinx.ext.mathjax')
```

Comment:
I choose `mathjax` as a default renderer. It is commonly used, lightweight and not depending on external tools.
And I also choose other math extensions are not loaded by default. So user should add one of them to extension list.